### DISCUSSION TITLE
Negative Media coefficient

### DISCUSSION THREAD

**Discussion by shubhambooking2** [pymc-labs/pymc-marketing#1232] [Q&A]
Hi ,

iam using TVP for modelling the time varying effects on media channels. I also have enforced the beta coefficient to be strictlly positive using HalfNormal Distribution, However in the inference data i see that some of the media contribution comes out to be negative. Can anyone suggest why this could be case (given the fact that iam using HalfNormal Beat Priors) and also i see the same situation for the ROAS, which iam suspecting is primarily since the contribution for these channels are negative.

i suspect this happening possibly because of TVP which can have negative values. if thats the reason, how we can enforce the TVP to be non-negative. Thanks

**Comment by williambdean** [comment]
Can you share the plot of the media_latent_multiplier variable
